### DISCUSSION TITLE
milvus单机版etcd启动失败

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#21774] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/117245915/212855140-4cd2dc46-9ef8-48e6-9716-9ed9549e1b97.png)
启动一直报错：panic: runtime error: slice bounds out of range [:8] with capacity 5
有大神知道解决方法吗，急！！

**Comment by LoveEachDay** [comment]
Seems your etcd have existing data in the data directory. 
Do you run milvus standalone with docker-compose?
 If so, you can remove the etcd data volume in the directory `volumes/etcd`.

**Reply by RongHouchao** [reply]
删除了etcd，集群里的数据能恢复吗？生产环境，比较急。
docker-compose重启也启动不了。

**Reply by RongHouchao** [reply]
我自己测试环境也搭建了一个，删除etcd容器后再启动，collection信息查不到了。
![image](https://user-images.githubusercontent.com/117245915/213059746-a65ab0e3-2e41-4e35-91d4-29bf779d1a19.png)

**Reply by LoveEachDay** [reply]
Yes, collection metadata is stored in etcd. If remove etcd data, all metadata lose. 
And you can see any collection info from attu.

**Reply by LoveEachDay** [reply]
> 删除了etcd，集群里的数据能恢复吗？生产环境，比较急。 docker-compose重启也启动不了。

You may not delete etcd data for a production environment.
